### PR TITLE
Add test to create a vnet

### PR DESCRIPTION
**Pull request by matheusvill**

### CODE REVIEW

File: tests/azure/vnet_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		f.Ctx,
+		t,
+		"./testdata/create_vnet.sh",
+		vnet,
+		f.ResGroupName,
+		f.Location,
+		"10.116.0.0/16",
+	)
+
+	vnets := azure.NewVnet(f.Ctx, t, f.Session, f.ResGroupName)
```
Comment (tiago4orion) [inline tests/azure/vnet_test.go]: calling it vnets is misleading

File: tests/lib/azure/id.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package azure
+
+import "fmt"
+
+func getID(resource string, method string, name string) string {
```
Comment (tiago4orion) [inline tests/lib/azure/id.go]: I thought `getID` was something different until I saw the implementation. I think this name is misleading also. Maybe something like `newID` or `makeID`.

File: tests/lib/azure/vnet.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	as.client.Authorizer = s.token
+	return as
+}
+
+// AssertExists checks if availability sets exists in the resource group.
+// Fail tests otherwise.
+func (vnet *Vnet) AssertExists(t *testing.T, name string) {
+	retrier.Run(vnet.ctx, t, getID("Vnet", "AssertExists", name), func() error {
+		_, err := vnet.client.Get(vnet.resgroup, name, "")
+		if err != nil {
```
Comment (tiago4orion) [inline tests/lib/azure/vnet.go]: delete?

File: tests/lib/azure/vnet.go [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+			return fmt.Errorf("resource %s should not exist", name)
+		}
+		return nil
+	})
+}
+
+// Delete the availability set
+func (vnet *Vnet) Delete(t *testing.T, name string) {
+	retrier.Run(vnet.ctx, t, getID("Vnet", "Delete", name), func() error {
+		c := make(chan struct{})
```
Comment (tiago4orion) [inline tests/lib/azure/vnet.go]: if no ones read c, it would not deadlock in the call below if something try to send some data?

**Comment by katcipis** [comment]
Closes #35

**Pull request merged**